bin/node/runtime/src/lib.rs

```diff
@@ -615,7 +615,47 @@ impl_runtime_apis! {
 			let mut batches = Vec::<BenchmarkBatch>::new();
 			let params = (&pallet, &benchmark, &lowest_range_values, &highest_range_values, &steps, repeat);
 
+			use pallet_bridge_currency_exchange::benchmarking::{
+				Module as BridgeCurrencyExchangeBench,
+				Trait as BridgeCurrencyExchangeTrait,
+				ProofParams as BridgeCurrencyExchangeProofParams,
+			};
+
+			impl BridgeCurrencyExchangeTrait for Runtime {
+				fn make_proof(
+					proof_params: BridgeCurrencyExchangeProofParams<AccountId>,
+				) -> crate::exchange::EthereumTransactionInclusionProof {
+					use sp_currency_exchange::DepositInto;
+
+					if proof_params.recipient_exists {
+						<Runtime as pallet_bridge_currency_exchange::Trait>::DepositInto::deposit_into(
+							proof_params.recipient.clone(),
+							ExistentialDeposit::get(),
+						).unwrap();
+					}
+
+					let transaction = crate::exchange::prepare_ethereum_transaction(
+						&proof_params.recipient,
+						|tx| {
+							// our runtime only supports transactions where data is exactly 32 bytes long
+							// => we are ignoring `transaction_size_factor` here
+							tx.value = (ExistentialDeposit::get() * 10).into();
+						},
+					);
+					let transactions = sp_std::iter::repeat(transaction.clone())
+						.take(1 + proof_params.proof_size_factor as usize)
+						.collect::<Vec<_>>();
+					let block_hash = crate::exchange::prepare_environment_for_claim::<Runtime>(&transactions);
+					crate::exchange::EthereumTransactionInclusionProof {
+						block: block_hash,
+						index: 0,
+						proof: transactions,
+					}
+				}
+			}
+
 			add_benchmark!(params, batches, b"bridge-eth-poa", BridgeEthPoA);
+			add_benchmark!(params, batches, b"bridge-currency-exchange", BridgeCurrencyExchangeBench::<Runtime>);
```

Comment:
Not relevant for this PR, but there was recently a change introduced in Substrate that changed the arguments to this macro. We'll need to update that next time we bump Substrate